packages/ui/components/scheduler/components/blocking/BlockedList.tsx

```diff
@@ -0,0 +1,111 @@
+import { useId, useMemo } from "react";
+
+import dayjs from "@calcom/dayjs";
+
+import { useSchedulerStore } from "../../state/store";
+import { BlockedTimeCell } from "./BlockedTimeCell";
+
+type Props = {
+  day: dayjs.Dayjs;
```

Comment:
Day is the current grid "DAY" we are looking at as this is iterated over in the main scheduler component